pack/tic/itd.json

```diff
@@ -84,6 +189,39 @@
 		"type_code": "asset",
 		"xp": 0
 	},
+	{
+		"code": "07119",
+		"cost": 3,
+		"deck_limit": 2,
+		"faction_code": "mystic",
+		"illustrator": "Patrick McEvoy",
+		"name": "Shroud of Shadows",
+		"pack_code": "itd",
+		"position": 119,
+		"quantity": 2,
+		"skill_agility": 1,
+		"slot": "Arcane",
+		"text": "Uses (3 charges).\n[Action] Spend 1 charge: <b>Evade.<b/> This evasion uses [willpower] instead of [agility]. If you succeed and the evaded enemy is non-[[Elite]], you may move that enemy to a connecting location. If a [curse] token is revealed during this evasion attempt, you may move to a connecting location or place 1 charge on Shroud of Shadows.",
+		"traits": "Spell. Cursed.",
+		"type_code": "asset",
+		"xp": 0
+	},
+	{
+		"code": "07120",
+		"cost": null,
+		"deck_limit": 1,
+		"faction_code": "mystic",
+		"illustrator": "Tiziano Baracchi",
+		"name": "Paradoxical Covenant",
+		"pack_code": "itd",
+		"permanent": true,
+		"position": 120,
+		"quantity": 1,
+		"text": "Permanent. Limit 1 [[Covenant]] per deck.\n[reaction] After an investigator at your location performs the \"reveal the chaos token\" step of a skill test, if both a [bless] and a [curse] token were revealed during that test, exhaust Paradoxical Covenant: This test automatically succeeds. <i>(Remove each [bless] and [curse] token revealed from the chaos bag after this test ends.)</i>",
```

Comment:
missing word: it should read 'if both a [bless] token and a [curse] token', repeating the word token.